core/dbt/parser/manifest.py

```diff
@@ -840,10 +840,13 @@ def skip_partial_parsing_because_of_macros(self):
         return False
 
     def read_manifest_for_partial_parse(self) -> Optional[Manifest]:
-        if not get_flags().PARTIAL_PARSE:
+        flags = get_flags()
+        if not flags.PARTIAL_PARSE:
             fire_event(PartialParsingNotEnabled())
             return None
-        path = os.path.join(self.root_project.project_target_path, PARTIAL_PARSE_FILE_NAME)
+        path = flags.partial_parse_file_path or os.path.join(
```

Comment:
nit: let's use upper-case variable names for referencing `flags` attributes for consistency (as we've done above in flags.PARTIAL_PARSE). so flags.PARTIAL_PARSE_FILE_PATH here